lib/modules/blockchain_process/gethClient.js

```diff
@@ -20,23 +63,33 @@ class GethCommands {
       cmd.push(`--datadir=${config.datadir}`);
     }
 
-    if (config.syncMode || config.syncmode) {
-      cmd.push("--syncmode=" + (config.syncmode || config.syncMode));
+    if (config.syncMode) {
+      cmd.push("--syncmode=" + config.syncMode);
```

Comment:
Are you sure? It is the only parameter with this peculiar treatment. There are others parameters of the Geth's CLI that are written in lowercase (rpcPort, networkId, etc.) but are in camelCase in the configuration file. Maybe with the introduction of multiclient support we can take the chance to get rid of multiple syntaxes for parameters in config files?

Reply:
Hum, yeah you're right. I personally prefer supporting what Geth does directly (lowercase), but since we already have support for camelCase, I think we should support both. That's just my opinion though. I'll see with my colleagues. In the meantime, leave it as is.

Reply:
Done. Please note I implemented the lowercase support in `lib/modules/blockchain_process/blockchain.js`. In the config object I keep a single parameter, even if multiple syntax are supported externally.